client/src/App/Components/List.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+
+class List extends Component {
```

Comment:
You should refactor this to be a functional component (like your Home component). You cannot use hooks inside a class based component